aws-lambda-layer/Dockerfile

```diff
@@ -20,7 +22,7 @@ RUN wget https://github.com/veracross/freetds/raw/master/freetds/freetds-${FREET
 RUN mkdir build
 
 RUN cd /var/task && \
-    mkdir lib && \
+    mkdir lib build && \
```

Comment:
1. shouldn't `build` already exist, since there's this `mkdir build` command above?
2. should we add `mkdir lib` to the list above? (And, should it all be in a single command rather than 5 `RUN mkdir ...` in a row?)
2. it occurs to me that `mkdir build` should probably be moved from this line, up to the list of dirs created?

Reply:
1. the first `mkdir build` is created in the home path not in the `/var/task` directory, the previous image `lambci/lambda` contained the directories `/var/task/lib` and `/var/task/build`, that's why I had to create it manually when we change the base image.

2. mmm, I don't think we even need the `lib` folder in `/var/task` we're only using `/var/task/build/` to save the library zip files.

3. yes, sounds good.